README.md

```diff
@@ -22,6 +22,12 @@ Think of it like [Tailwind](https://tailwindcss.com/) for JavaScript.
 
 That's it. It will initialize itself.
 
+For production environments, it's recommend linking to a specific version number to avoid unexpected breakage from newer versions. 
```

Comment:
maybe?
```suggestion
For production environments, it's recommended to pin a specific version number in the link to avoid unexpected breakage from newer versions. 
```